robottelo/cli/base.py

```diff
@@ -37,8 +38,26 @@ def __str__(self):
 
     def __repr__(self):
         """Include return_code, stderr and msg to improve logging"""
-        tmpl = u'CLIReturnCodeError(return_code={!r}, stderr={!r}, msg={!r}'
-        return tmpl.format(self.return_code, self.stderr, self.msg)
+        return u'{}(return_code={!r}, stderr={!r}, msg={!r}'.format(
```

Comment:
Since the returned artifact has included one more element (`type(self).__name__`), the docstring needs to be updated :)